### PR TITLE
Adding `tbl_ae_focus()`

### LINKED ISSUE #28: Feature Request: Add ability to summarize AEs by "Any Grade" and  "Grade 3+", etc
I often include tables like the one below in my manuscripts.

It can fit in quite easily into `tbl_adverse_events()`, but needs quite a bit of customization to make it work. Perhaps best to write a small wrapper for it?

image

### PR DESCRIPTION
**Pull request by ddsjoberg**
- Adding function `tbl_ae_focus()` 
- I don't love the name, but can be addressed later 🤷🏼 
closes #28

### CODE REVIEW

**Comment by shannonpileggi** [comment]
Ha, regarding the name!  Yes, it isn't quite binary.  More like...

`tbl_ae_subset()`
`tbl_ae_zoom()`
`tbl_ae_compare()`
`tbl_ae_focus()`

**Comment by ddsjoberg** [comment]
> I want to make sure I understand how you are thinking about the word `binary`, and what are the expectations for the `include` variable. 

Yep, all variables passed in `include=` must be logical/binary.

**Comment by ddsjoberg** [comment]
I'll update the function name to `tbl_ae_focus()`! I prefer it to the current name for sure

File: R/tbl_ae_focus.R [hunk 1]
```diff
@@ -0,0 +1,251 @@
+#' Tabulate AE Binary Summaries
```
Comment (shannonpileggi) [inline R/tbl_ae_focus.R:1]:
```suggestion
#' Tabulate AE Focused (Dichotomous) Summaries
```

File: R/tbl_ae_focus.R [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+#'
+#' Summarize AE data that is binary or dichotomous. For example, report the
```
Comment (shannonpileggi) [inline R/tbl_ae_focus.R:3]:
```suggestion
#' Summarize dichotomous AE data. For example, report the
```

**Review by shannonpileggi — CHANGES_REQUESTED** [review]
This function is awesome! It will allow so many custom tables!

**Review by shannonpileggi — COMMENTED** [review]
Go ahead and merge or update tests / examples with updated data first?

**Pull request merged**